manifests/kube/kube@0.2.0.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "kube",
+    "description": "A plugin to manage Spin apps in Kubernetes",
+    "homepage": "https://github.com/spinkube/spin-plugin-kube",
+    "version": "0.2.0",
```

Comment:
Now that kube.json is updated with 0.2.0, should this file rather be `kube@0.1.1.json` with the 0.1.1 data?

Reply:
Ah! Yes. Should be fixed now. Thank you.